curl/curl.go

```diff
@@ -104,16 +124,17 @@ func (c *Curl) Reset() {
 	}
 }
 
-// HashTrits returns hash of the given trits.
-func HashTrits(trits Trits) Trits {
+// HashTrits returns hash trits of the given trits.
+func HashTrits(trits Trits) (Trits, error) {
 	c := NewCurl()
-	c.Absorb(MustTritsToTrytes(trits))
-	return MustTrytesToTrits(c.Squeeze())
+	c.Absorb(trits)
+	return c.Squeeze(HashTrinarySize)
 }
 
-// HashTrytes returns hash of t.
+// HashTrytes returns hash trytes of the given trytes.
+// It panics if the given trytes are not valid.
```

Comment:
Functions which can panic must be prefix with "Must". Consider changing the source function which can panic to return an error.